docs/reference/supported-environments.md

```diff
@@ -33,7 +39,36 @@ For example, 1.2+ means support for the minor version 2, and any higher minors (
 
 ## Camunda 8 Self-Managed
 
-We highly recommend running Camunda 8 Self-Managed in a Kubernetes environment. We provide officially supported [Helm Charts](/self-managed/platform-deployment/helm-kubernetes/overview.md) for this. Please follow the [Installation Guide](/self-managed/platform-deployment/overview.md) to learn more about installation possibilities.
+### Deployment options
+
+- [Helm/Kubernetes](/self-managed/platform-deployment/helm-kubernetes/overview.md) [recommended]
+  - [Platforms](/self-managed/platform-deployment/helm-kubernetes/platforms/platforms.md)
+    - [Amazon EKS](/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/amazon-eks.md)
+    - [Microsoft AKS](/self-managed/platform-deployment/helm-kubernetes/platforms/microsoft-aks.md)
+    - [Google GKE](/self-managed/platform-deployment/helm-kubernetes/platforms/google-gke.md)
+    - [Red Hat OpenShift](/self-managed/platform-deployment/helm-kubernetes/platforms/redhat-openshift.md) (4.11+)
+- [Docker](/self-managed/platform-deployment/docker.md)
+- [Manual](/self-managed/platform-deployment/manual.md)
+
+Ensure the Camunda component versions are compatible with the Helm chart version as defined in the [matrix](https://helm.camunda.io/camunda-platform/version-matrix/).
```

Comment:
This statement is only valid for the "HELM" option, yet it reads and is placed as if it would apply to the entire section. Can you please check whether we cannot put it under the - HELM list point?

Reply:
I moved this to an admonition titled "Helm chart compatibility".

Adding it to the list would make for a disjointed list.